scripts/deploy.sh

```diff
@@ -1,11 +1,16 @@
 echo "Extension layer"
+rm -rf extensions extension-python-modules extensions.zip runtime.zip python-runtime __MACOSX
 echo "-Installing the package and its dependencies"
 python -W ignore setup.py --quiet bdist_wheel  # create .egg-info
-enc_location=../common-resources/encrypted_files/credentials_production.enc
-mkdir -p ~/.aws
-echo ${KEY} | gpg --batch -d --passphrase-fd 0 ${enc_location} > ~/.aws/credentials
-unset AWS_ACCESS_KEY_ID
-unset AWS_SECRET_ACCESS_KEY
+
+if [ -n "$CI" ]; then
+  echo "- Use credentials_production"
+  enc_location=../common-resources/encrypted_files/credentials_production.enc
+  mkdir -p ~/.aws
+  echo ${KEY} | gpg --batch -d --passphrase-fd 0 ${enc_location} > ~/.aws/credentials
+  unset AWS_ACCESS_KEY_ID
+  unset AWS_SECRET_ACCESS_KEY
+fi
```

Comment:
please add else clauses to log (when running locally) that these steps are being skipped because it's being run locally